### PR TITLE
Add full text specs for Hungarian

### PR DESCRIPTION
**Pull request by hannaw93**
## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Adds Hungarian texts and specs to check the assessments functionality for Hungarian.

## Relevant technical choices:
*

## Test instructions

This PR can be tested by following these steps:
* Run yarn test and make sure all tests pass

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes
* [ ] This PR changes the UI in the plugin. I have added the 'UI change' label to this PR.

## Quality assurance

* [x] I have tested this code to the best of my abilities
* [x] I have added unittests to verify the code works as intended

Fixes https://yoast.atlassian.net/browse/LINGO-607

### CODE REVIEW

File: packages/yoastseo/spec/fullTextTests/testTexts/hu/hungarianPaper1.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+const name = "hungarianPaper1";
+
+const paper = new Paper( content, {
+	keyword: "Világegyetem",
+	synonyms: "világegyetem",
+	description: "A világegyetem (latinosan univerzum) csillagászati fogalom, minden létező összességét jelenti.",
+	title: "Világegyetem",
+	titleWidth: 450,
+	locale: "tr_TR",
```
Comment (iolandasequino) [inline packages/yoastseo/spec/fullTextTests/testTexts/hu/hungarianPaper1.js:12]: The locale should be "hu_HU"

**Review by iolandasequino — CHANGES_REQUESTED** [review]
CR and Acceptance: 👎 
Yarn test will have to re-run after the changes to the locale

File: packages/yoastseo/spec/fullTextTests/testTexts/hu/hungarianPaper1.js (hunk 1 above)
Comment (hannaw93) [reply packages/yoastseo/spec/fullTextTests/testTexts/hu/hungarianPaper1.js]: Thanks, fixed!

**Comment by iolandasequino** [comment]
CR & Acceptance: 👍

**Pull request merged**